app/src/main/java/net/gsantner/opoc/format/todotxt/SttCommander.java

```diff
@@ -62,6 +62,7 @@
     public static final Pattern PATTERN_PRIORITY_F = Pattern.compile("(?:^|\\n)\\(([Ff])\\)\\s");
     public static final Pattern PATTERN_COMPLETION_DATE = Pattern.compile("(?:^|\\n)(?:[Xx] )(" + PT_DATE + ")");
     public static final Pattern PATTERN_CREATION_DATE = Pattern.compile("(?:^|\\n)(?:\\([A-Za-z]\\)\\s)?(?:[Xx] " + PT_DATE + " )?(" + PT_DATE + ")");
+    public static final Pattern PATTERN_IS_DATE = Pattern.compile(PT_DATE);
```

Comment:
The pattern catches a date, `_IS_` is something that is done by using the respective methods. Suggest to rename to just PATTERN_DATE